modules/geo-layers/src/terrain-layer/terrain-layer.js

```diff
@@ -165,7 +170,7 @@ export default class TerrainLayer extends CompositeLayer {
     }
 
     const {zRange} = this.state;
-    const ranges = data.map(arr => {
+    const ranges = data.filter(arr => arr).map(arr => {
```

Comment:
I had to add this filter because there can now be `null`s, in the list, and `arr[0]` will fail.

Reply:
`filter(Boolean)`

Reply:
So does that mean I can use just `filter(arr)` or `filter(Boolean(arr))`?

Reply:
`arr.filter(Boolean)`. `Boolean` is a function.